data/Sri Guru Granth Sahib Ji/1394.json

```diff
@@ -800,7 +800,7 @@
         "pronunciation_information": "",
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "iq nr dulIcY bhih; iq nr auQip ibQpih ]"
+          "SGPC": "iqnr dulIcY bhih; iqnr auQip ibQpih ]"
```

Comment:
```suggestion
          "SGPC": "iqnr dulIcY bhih; iq nr auQip ibQpih ]"
```

Reply:
This is wrong according to SGPC Shabadarth, we aren't using Dr. Thind's PDF as proof.

Reply:
This suggestion comes from the image referenced in #1565